manifests/e/ElectronicArts/EADesktop/12.0.192.5129/ElectronicArts.EADesktop.installer.yaml

```
@@ -1,24 +1,24 @@
-# Automatically updated by the winget bot at 2022/Mar/08
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
-
-PackageIdentifier: ElectronicArts.EADesktop
-PackageVersion: 12.0.188.5117
-Platform:
-- Windows.Desktop
-MinimumOSVersion: 10.0.0.0
-InstallerType: burn
-Scope: machine
-InstallModes:
-- interactive
-- silent
-- silentWithProgress
-InstallerSwitches:
-  Custom: ACCEPT=YES
-  Upgrade: /upgrade
-UpgradeBehavior: install
-Installers:
-- Architecture: x64
-  InstallerUrl: https://origin-a.akamaihd.net/EA-Desktop-Client-Download/installer-releases/EAappInstaller.exe
-  InstallerSha256: 6F34B2A9D1F03723E698B7417180C43C452C6C98358F524AC34AD5E4A28FF21B
-ManifestType: installer
-ManifestVersion: 1.1.0
+# Created with YamlCreate.ps1 v2.1.2 $debug=AUSU.7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: ElectronicArts.EADesktop
+PackageVersion: 12.0.192.5129
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: burn
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Upgrade: /upgrade
+  Custom: ACCEPT=YES
```

```suggestion
  Custom: /install ACCEPT=YES
```
It does appear like /upgrade actually worked as expected, for whoever is looking at the PR, do you think /install would work as expected?
Because this issue appears at random times: https://github.com/microsoft/winget-pkgs/pull/55247#event-6279421370